cfpb/cfgov-refresh
# 282-fix-sourcemaps

Adds Sourcemaps and removes CSSMIN task

## Additions

- Creates sourcemap file (thinking this should be git ignored and devs just make sure they run grunt upon pulling from the repo, otherwise it's just another giant file in your diff)

## Removals

- CSSMIN from package.json and Gruntfile

## Changes

- Added sourcemap options to `less` task

## Testing

- fetch and run `grunt` or `grunt:cssdev`
- open project locally and inspect any elem
- check source file in upper right of style tab (should be a Less file)
- mindblown

![tim-and-eric-mind-blown](https://cloud.githubusercontent.com/assets/1280430/6398578/8eb79e1c-bdba-11e4-8832-73ebe6c5d05b.gif)


## Review

Including all FEWDs because this could really be used Agency wide and it's worth a discussion (sorry if I forgot anyone):
- @anselmbradford 
- @sebworks 
- @Scotchester 
- @virginiacc 
- @sephcoster 
- @cfarm
- @kimberlymunoz
- @contolini
- @ascott1 
- @mistergone

## Preview

![screen shot 2015-02-26 at 1 08 09 pm](https://cloud.githubusercontent.com/assets/1280430/6398403/531c0b82-bdb9-11e4-9e30-fbed05ffa075.png)


## Notes

- Sourcemaps are incredibly important when tracking down styles in compiled less files
- CSSMIN and other minification tools remove the sourcemap reference from the css file
- Minification should really happen in the deployment process, not development process
- Fixes #282
